{"name": "test_invalid_login", "status": "passed", "start": 1584513557334, "stop": 1584513561335, "uuid": "119a76c4-dd24-4eb0-a79e-5ca661e09f5b", "historyId": "513d84f1856f15388337a423d09ebd92", "testCaseId": "55f35891554f4a3a55efb423475adf20", "fullName": "tests.test_invalid_login#test_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_invalid_login"}, {"name": "host", "value": "Bigpoe.local"}, {"name": "thread", "value": "43819-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_invalid_login"}]}